pkg/model/trace/matches.go

```diff
@@ -9,10 +9,26 @@ import (
 	v1common "github.com/grafana/tempo/pkg/tempopb/common/v1"
 	v1 "github.com/grafana/tempo/pkg/tempopb/trace/v1"
 	"github.com/grafana/tempo/pkg/util"
-	"github.com/grafana/tempo/tempodb/search"
 )
 
-const RootSpanNotYetReceivedText = "<root span not yet received>"
+const (
```

Comment:
i feel like ultimately these will end up somewhere else. other block types like columnar will need access to the same constants?

Reply:
`pkg/model/trace` seems like an ok location, and I'd expect other block types to reference it for common operations like translation/inspection of proto. Were you thinking a different .go in this folder since, I agree, other block types wouldn't be using `matches.go` ?

Reply:
I guess in my mind `pkg/model` is eventually going away since it works with the "byte slice" tempodb style. Although maybe it lives on for constants like this? Also, it's possible that the "SegmentDecoder" functionality survives new block formats as well.

0 issues with moving this here now. I think this is all in flux and I'm sure everything will land somewhere that makes sense.